test/utils/test_display.jl

```diff
@@ -0,0 +1,40 @@
+include("../../src/utils/text_display.jl")
```

Comment:
This will break depending on how tests are run. We can't assume relative paths will always work, or rely on tests being invoked from the same initial path every time.

If this is absolutely necessary, use `pkgdir(ADRIA)` instead:

```julia
joinpath(pkgdir(ADRIA), "src", "utils", "text_display.jl")
```

Reply:
I also have a conceptual issue with the file name `test_display`.

It already is in a directory called `test` so naturally everything in it will be related to tests.
Best to name it after what it is testing.

Reply:
I.. don't think I need to include anything at all, let me test removing that to see what happens.